### PR TITLE
Food Model Migration and Controller Implemented

### PR DESCRIPTION
**Pull request by peterdgreat**
- Implemented Food Model  Migration

- Created Controller For Food Index Create new 

- Started Out With Food View

### CODE REVIEW

File: app/views/foods/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div>
+<% @foods.each do |food| %>
```
Comment (Wikitelmex) [inline app/views/foods/index.html.erb]: [ ] kindly add bootstrap styling to this loop

File: app/views/foods/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<%= form_with model: @food, url: foods_path do |form| %>
```
Comment (Wikitelmex) [inline app/views/foods/new.html.erb:3]: [] kindly add bootstrap styling to this loop

**Review by Wikitelmex — CHANGES_REQUESTED** [review]
GREAT JOB WORKING ON FOODS!!!

However, there's left some changes you need to implement

Summary:

- [ ] Optional: Implement bootstrap styling to the views you make (strongly suggested)
- [ ] Critical: change price from integer to float, because prices usually use cents

File: app/views/foods/index.html.erb (hunk 1 above)
Comment (peterdgreat) [reply app/views/foods/index.html.erb]: Yeah I never started the styling, I only requested because you asked...

**Review by Wikitelmex — APPROVED** [review]
GREAT JOB!!!

There's nothing left to say, so it's time to merge it!!!!

**Pull request merged**